### PR TITLE
Doc(EJ2-849633): UG for OnDemand Checkbox/Excel Filter type

### PR DESCRIPTION
**Pull request by vasanthakumarsf3389**
Ug for Excel or Checkbox filtering with on-demand feature

### CODE REVIEW

File: ej2-javascript/code-snippet/grid/checkbox-excel-filter/es5-datasource.js [hunk 1]
```diff
@@ -0,0 +1,592 @@
+var data = [
+    {
+        OrderID: 10248, CustomerID: 'VINET', EmployeeID: 5, OrderDate: new Date(8364186e5),
```
Comment (MaithiliyK) [inline ej2-javascript/code-snippet/grid/checkbox-excel-filter/es5-datasource.js:3]: remove unwanted data source

File: ej2-javascript/code-snippet/grid/checkbox-excel-filter/datasource.ts [hunk 2]
```diff
@@ -0,0 +1,719 @@
...
+        ShipName: 'Que Delícia', ShipCity: 'Rio de Janeiro', ShipAddress: 'Rua da Panificadora, 12',
+        ShipRegion: 'RJ', ShipPostalCode: '02389-673', ShipCountry: 'Brazil', Freight: 3.05, Verified: !1
+    },
+    {
+        OrderID: 10262, CustomerID: 'RATTC', EmployeeID: 8, OrderDate: new Date(8379738e5),
+        ShipName: 'Rattlesnake Canyon Grocery', ShipCity: 'Albuquerque', ShipAddress: '2817 Milton Dr.',
+        ShipRegion: 'NM', ShipPostalCode: '87110', ShipCountry: 'USA', Freight: 48.29, Verified: !0
+    }];
+
+export let employeeData: Object[] = [{
```
Comment (MaithiliyK) [inline ej2-javascript/code-snippet/grid/checkbox-excel-filter/datasource.ts:78]: from all the datasource too

Comment (vasanthakumarsf3389) [reply ej2-javascript/code-snippet/grid/checkbox-excel-filter/datasource.ts]: resolved - removed unwanted datasource

File: ej2-javascript/code-snippet/grid/checkbox-excel-filter/es5-datasource.js (hunk 1 above)
Comment (vasanthakumarsf3389) [reply ej2-javascript/code-snippet/grid/checkbox-excel-filter/es5-datasource.js]: resolved - removed unwanted datasource

**Review by MaithiliyK — APPROVED** [review]
changes are fine

**Pull request merged**